docs/tool/index.html

```diff
@@ -65,6 +66,27 @@
 
     <div id="body-wrapper"></div>
 <div id="loading-state-screen">Loading saved state . . .</div>
+<!-- Modal -->
+  <div class="modal fade" id="myModal" role="dialog">
```

Comment:
You should change the `id` of this modal to something more specific (like id="exportDataModal" or something). You'll need to change it both here and in the data-target of the button that activates it.